<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageSelect</class>
 <widget class="QDialog" name="ImageSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>351</width>
     <height>162</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_1">
      <item>
       <widget class="QRadioButton" name="radioButton_1">
        <property name="text">
         <string>Start ID / End ID</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_start_1">
        <property name="text">
         <string>Start ID :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_start_1"/>
      </item>
      <item>
       <widget class="QLabel" name="lbl_end_1">
        <property name="text">
         <string>End ID :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_end_1"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>Start ID / Num of Images</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_start_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Start ID:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_start_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_num_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Num :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_num_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="lbl_message">
      <property name="maximumSize">
       <size>
        <width>9999</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ImageSelect</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ImageSelect</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
